### PR TITLE
Added quick tip: Creating themes with shopware cli tools

### PR DESCRIPTION
**Pull request by tobiasberge**

### CODE REVIEW

**Comment by tiagojsag** [comment]
There's already an article covering the CLi: https://devdocs.shopware.com/developers-guide/shopware-5-cli-commands/

IMO it makes sense to somehow group the CLI commands. My suggestion would be to add your text to the existing CLI doc, and link it from the relevant guides in the "designers guide" section.

@bcremer your opinion?

**Comment by bcremer** [comment]
I find it ok to have a article covering the CLI-Tools in genral and a separate more detailed article on a specific command.

**Comment by bcremer** [comment]
In my opinion the current change is useless.
Most of this information can be found using the `-h` or `--help` option of the command:

```
$ ./bin/console sw:theme:create -h
Usage:
 sw:theme:create [--description="..."] [--author="..."] [--license="..."] parent template name

Arguments:
 parent                Name of the theme which should be extended.
 template              Name of the theme directory.
 name                  Name of the theme readable in theme manager.

Options:
 --description         Description of the theme to be created.
 --author              Author of the theme to be created.
 --license             Licence of the theme to be created.
```

So instead of introducing a high maintenance section in the documentation the documentation in the command should be improved, for example the descripton: 
https://github.com/shopware/shopware/blob/5.0/engine/Shopware/Commands/ThemeCreateCommand.php#L53

File: source/developers-guide/shopware-5-cli-commands/index.md [hunk 1]
```diff
@@ -65,3 +65,27 @@ Loads snippets from the `snippets` folder of your Shopware installation and crea
 - `include-plugins`: If set, active plugin snippets will also be exported.
 
 - `update`: By default, the generated SQL script only performs inserts. If the `update` option is provided, it will also handle update scenarios when duplicates are found. If the existing database snippet has `dirty` = 0, the value will be overwritten. If `dirty` is 1, it's not changed. Please note that, for a large number of snippets, enabling update support will make the SQL statements significantly slower to execute upon importing.
+
+## Themes
+
+### sw:theme:create
```
Comment (bcremer) [inline source/developers-guide/shopware-5-cli-commands/index.md]: That section just duplicates what is already documented in the command itself. Please see my comment in the conversation view.

**Pull request merged**